lib/msal-react/src/MsalContext.tsx

```diff
@@ -19,6 +20,7 @@ export interface IMsalContext {
 const defaultMsalContext: IMsalContext = {
     instance: stubbedPublicClientApplication,
     loginInProgress: false,
+    interactionInProgress: false,
```

Comment:
Might consider a single value whose value is an enum instead of multiple booleans.

Reply:
Added. Thanks for the suggestion, I like this better!